uaclient/api/u/pro/security/fix/common/__init__.py

```diff
@@ -0,0 +1,47 @@
+from typing import Optional
```

Comment:
A nit but I suggest changing the folder name from `common` to `_common` if that's allowed, to signal that it is for internal use only

Reply:
I though about this, but once we move the plan logic there as well, we will import some `DataObjects` on the cli module. So it not something that will be only used internally on the API. But if you think the rename is still worth it, I can do it. No problem